### ISSUE TITLE
Setup the EKS Cluster in AWS and deploy the beckn-onix on it

### ISSUE THREAD

**Issue by viraj89** [beckn/beckn-onix#194]
## Description
Set up the EKS cluster in the AWS account provided by the AWS Team and deploy the `beckn-onix` on that EKS Cluster. Once the deployment is completed, share the Kubernetes manifest files with the AWS team.

## Goals
- [ ] Set up the EKS cluster in the designated AWS account.
- [ ] Deploy the `beckn-onix` application on the EKS cluster.
- [ ] Share the Kubernetes manifest files with the AWS team.

## Expected Outcome
- The EKS cluster is successfully set up in the specified AWS account.
- The `beckn-onix` application is deployed and functioning on the EKS cluster.
- Kubernetes manifest files are provided to the AWS team.

## Acceptance Criteria
- [ ] EKS cluster is created in the specified AWS account.
- [ ] `beckn-onix` is deployed and operational on the EKS cluster.
- [ ] Kubernetes manifest files are shared with the AWS team.

## Mockups / Wireframes
NA

## Product Name
Beckn-Onix

## Domain
DevOps

## Tech Skills Needed
AWS, EKS, Kubernetes, DevOps

## Complexity
Medium

## Category
Deployment

## Sub Category
Infrastructure Setup

**Comment by prasad-takale-eminds** [comment]
We have set up the EKS cluster in the AWS account provided by Moz. Here are the steps we followed to complete the setup:

1. Created a new IAM role specifically for the cluster setup.
2. Set up a new public subnet.
3. Created a new Internet Gateway and NAT Gateway.
4. Established the new EKS cluster.
5. Created a new IAM role for the node group creation.
6. Configured the node group for the EKS cluster.
